src/main/resources/org/jenkinsci/plugins/gwt/global/JobFinderConfig/config.jelly

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core"
+         xmlns:d="jelly:define"
+         xmlns:f="/lib/form"
+         xmlns:l="/lib/layout"
+         xmlns:st="jelly:stapler"
+         xmlns:t="/lib/hudson"
+         xmlns:c="/lib/credentials">
+
+  <f:section title="Generic Webhook Trigger Job Finder Selector">
+    <f:optionalBlock field="enabled" title="Precise Job Finder enabled" inline="true">
```

Comment:
I dont see the need for this boolean. You should be able to just check if the list is empty or not. If not empty it is enabled, if empty it is disabled.

Reply:
I think this boolean is distinct to the precise job finder and old job finder. Convenient for subsequent expansion of job finder，There have the other job finder In the future.